src/ontology/mondo-edit.obo

```diff
@@ -256773,27 +256775,21 @@ property_value: confidence "3.012345679012345" xsd:double
 [Term]
 id: MONDO:0013082
 name: Hirschsprung disease-ganglioneuroblastoma syndrome
+def: "A rare, genetic, developmental defect during embryogenesis syndrome characterized by total or partial colonic aganglionosis associated with peripheral, usually multifocal, neuroblastic tumors (ganglioneuroblastoma, neuroblastoma, ganglioneuroma). Congenital central hypoventilation syndrome, with variable severity of respiratory compromise, cardiovascular and ophthalmologic symptoms, consistent with autonomic nervous system dysfunction, is occasionally associated." [Orphanet:2151]
 subset: ordo_malformation_syndrome {source="Orphanet:2151"}
 subset: predisposition
 synonym: "Hirschsprung disease ganglioneuroblastoma" RELATED [GARD:0002695]
-synonym: "NBLST2" RELATED ABBREVIATION [MONDO:Lexical, OMIM:613013]
 synonym: "neuroblastoma with Hirschsprung disease" RELATED [OMIM:613013]
-synonym: "neuroblastoma, susceptibility to, 2" RELATED [MONDO:Lexical, OMIM:613013]
-synonym: "neuroblastoma, susceptibility to, type 2" EXACT [MONDORULE:1, OMIM:613013]
-synonym: "susceptibility to neuroblastoma 2" RELATED [OMIM:613013]
 xref: GARD:0002695 {source="shared-umls-xref", source="MONDO:equivalentTo"}
 xref: ICD10:Q43.1 {source="Orphanet:2151", source="ORDO:2151/attributed", source="ORDO:2151/ntbt"}
 xref: MESH:C538119 {source="MONDO:equivalentTo", source="Orphanet:2151", source="ORDO:2151/e"}
-xref: OMIM:613013 {source="MONDO:equivalentTo"}
 xref: Orphanet:2151 {source="MONDO:equivalentTo"}
-xref: Orphanet:635 {source="MONDO:subClassOf", source="OMIM:613013"}
+xref: Orphanet:635 {source="OMIM:613013", source="MONDO:relatedTo"}
```

Comment:
@sabrinatoro fyi- I changed this xref to relatedTo, it was previously subclassOf